test/uk/gov/hmrc/gform/graph/DependencyGraphSuite.scala

```diff
@@ -0,0 +1,176 @@
+/*
```

Comment:
we have a DependencyGraphSpec class already. If we plan to use msuite, we need to migrate all tests to the new structure some day

Reply:
> we have a DependencyGraphSpec class already

Yes I know, unfortunately the nature of it isn't exactly what I need and rather than bloat it decided ti create independent Suite.
We may let scala-test and munit to live side by side imho.